app-tracking-protection/vpn-api/src/main/java/com/duckduckgo/mobile/android/vpn/feature/AppTpSetting.kt

```diff
@@ -25,6 +25,7 @@ enum class AppTpSetting(override val value: String, override val defaultValue: B
     AlwaysSetDNS("alwaysSetDNS"),
     CPUMonitoring("cpuMonitoring"),
     ProtectGames("protectGames"),
+    ExceptionLists("exceptionLists", defaultValue = true),
```

Comment:
Why the default value set to `true`? This is not a feature that will/can be disabled

Reply:
Right, there should be no way to disable it. But why not be safe and set the default to `true`?